### ISSUE TITLE
`go get` deprecated on versions 1.18.x

### ISSUE THREAD

**Issue by ashleyconnor** [kennyp/asdf-golang#68]
**Describe the bug**
Looks like `go get` is [deprecated](https://go.dev/doc/go-get-install-deprecation) in versions `1.18.x` which breaks the default packages feature of this plugin.

**To Reproduce**
1. `echo 'golang.org/x/tools/gopls' >>  $HOME/.default-golang-pkgs`
2. `asdf install golang 1.18beta2`
3. See error below

```
Installing golang.org/x/tools/gopls go pkg... go: go.mod file not found in current directory or any parent directory.
	'go get' is no longer supported outside a module.
	To build and install a command, use 'go install' with a version,
	like 'go install example.com/cmd@latest'
	For more information, see https://golang.org/doc/go-get-install-deprecation
	or run 'go help get' or 'go help install'.
FAIL
```

**Expected behavior**
`gopls` to be installed.

**Desktop (please complete the following information):**
 - OS: OSX
 - Browser N/A
 - Version 12.1

**Additional context**
`asdf v0.7.8-4a3e3d6`

Thank you for writing this plugin ❤️

**Comment by kennyp** [comment]
Hey @ashleyconnor there's already a PR in to fix this #67, but I was just waiting on the author to validate my change to their original PR before merging. I'll go ahead and merge it and you can let me know if you have any more issues.

**Comment by ashleyconnor** [comment]
Tested and working. Thanks @kennyp and @jm96441n!
